app-backend/batch/src/main/scala/com/azavea/rf/batch/ingest/spark/Ingest.scala

```diff
@@ -323,6 +326,13 @@ object Ingest extends SparkJob with LazyLogging with Config {
     implicit def asS3Payload(status: IngestStatus): String = S3IngestStatus(sceneId, status).asJson.noSpaces
 
     try {
+      Integer.valueOf(Properties.envOrElse("AWS_BATCH_JOB_ATTEMPT", "0")) match {
```

Comment:
Based on the AWS docs, it seems like they make the first attempt default to 1:

> At runtime, the `AWS_BATCH_JOB_ATTEMPT` environment variable is set to the container's corresponding job attempt number. The first attempt is numbered 1, and subsequent attempts are in ascending order (2, 3, 4, and so on).

I don't know that changing it from 0 to 1 fixes anything, but figure it would be worth keeping in sync.

Reply:
The 0 was to indicate local usage, but I don't know that it serves any specific purpose, so I'll change.